django/core/checks/security/base.py

```diff
@@ -119,6 +122,13 @@
     id='security.E023',
 )
 
+E024 = Error(
+    'You have set the SECURE_CROSS_ORIGIN_OPENER_POLICY setting to an invalid '
+    'value.',
+    hint='Valid values are: {}.'.format(', '.join(sorted(CROSS_ORIGIN_OPENER_POLICY_VALUES))),
```

Comment:
can't we use f string here?

Reply:
Only if we use another variable, from the guide:

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style

> As a guide, f-strings should use only plain variable and property access, with prior local variable assignment for more complex cases

I think it's fine as-is